index.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="./sytles/mobile-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/tablet-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/destokp-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/slideshow-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/slideshow-stylesheet.css" />
+    <link rel="stylesheet" href="./sytles/overlay.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel='stylesheet' type='text/css'>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+
+</head>
+
+<body>
+    <header>
+        <div class="title">
+            <h1>The Ladies Gamer</h1>
+        </div>
+    </header>
+    <hr>
+
+    <!-- NAVIGATION BAR ALL DEVICE -->
+
+    <!-- Navigation Mobile -->
+    <nav class="nav-bar-mobile">
+        <div id="wrapper-botton">
+            <a class="button" href="#modal">JUMP TO</a>
+        </div>
+
+        <div id="modal" class="overlay">
+            <a class="cancel" href="#"></a>
+            <div class="popup">
+                <hr>
+                <h2>Filter by topic</h2>
+                <hr>
+                <a class="close" href="#">&times;</a>
+                <div class="menu-content">
+                    <p>Reviews</p>
+                    <p>Youtubers</p>
+                    <p>Guides</p>
+                    <p>Deals</p>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <!-- Navigation Tablet -->
+    <nav class="nav-bar-tablet">
+        <div class="nav-options">
+            <a href="">Reviews</a>
+            <a href="">Youtubers</a>
+        </div>
+        <div class="nav-image">
+            <img src="./img/mario-star-coin-.png">
+        </div>
+        <div class="nav-options">
+            <a href="">Guides</a>
+            <a href="">Deals</a>
+        </div>
+    </nav>
+
+
+    <!-- Navigation Destokp -->
+    <nav class="nav-bar-destokp">
+        <a href="">Reviews</a>
+        <a href="">Youtubers</a>
+        <a href="">Guides</a>
+        <a href="">Deals</a>
+    </nav>
```

Comment:
Instead of duplicated the content (since it is the same content) we could use the CSS to style wether or not an element should be visible and also position it.

Reply:
Thank you I will try it for the next project!